spdx-utils/src/test/kotlin/SpdxExpressionTest.kt

```diff
@@ -336,5 +336,20 @@ class SpdxExpressionTest : WordSpec() {
                 spdxExpression.validChoices() shouldContainExactlyInAnyOrder choices
             }
         }
+
+        "offers choice" should {
+            "return true if the expression contains the OR operator" {
+                "a OR b".parse().offersChoice() shouldBe true
+                "a AND b OR c".parse().offersChoice() shouldBe true
+                "a OR b AND c".parse().offersChoice() shouldBe true
+                "a AND b AND c OR d".parse().offersChoice() shouldBe true
+            }
+
+            "return false if the expression does not contain the OR operator" {
+                "a".parse().offersChoice() shouldBe false
```

Comment:
How about `"a OR a".parse().offersChoice()` ?

Reply:
It would currently return true, because the DNF does not eliminate duplicates. I have an idea how to change the algorithm to fix this, but I'd like to do it in a separate PR after this is merged, probably together with the normalization improvement I mentioned earlier.